Source/DafnyCore/AST/MemberDecls.cs

```diff
@@ -105,6 +106,19 @@ public MemberDecl(RangeToken rangeToken, Name name, bool hasStaticKeyword, bool
   }
 
   public virtual IEnumerable<Expression> SubExpressions => Enumerable.Empty<Expression>();
+
+  public bool HasOnlyAttribute() {
```

Comment:
It's a little unfortunate to have this implemented twice, once on members and once on statements, but it might be worse to lift it up to arbitrary AST nodes, so perhaps it's reasonable to keep it this way.

Reply:
Yeah, until we have mix-ins :-)

Reply:
Actually with extensions we have mixins. Let me refactor it.